warehouse/query-core/src/main/java/datawave/query/index/lookup/CreateUidsIterator.java

```diff
@@ -62,8 +70,10 @@
     private static final Logger log = Logger.getLogger(CreateUidsIterator.class);
     
     public static final String COLLAPSE_UIDS = "index.lookup.collapse";
+    public static final String PARSE_ROOT_UIDS = "index.lookup.parse.root.uids";
```

Comment:
This turns TLD parsing on or off, right?  So why not name it TLD... something?

Reply:
I can change that to PARSE_TLD_UIDS